extensions/laravel-forge/src/Site.tsx

```diff
@@ -81,6 +81,18 @@ export const SitesSingleView = ({ site, server }: { site: ISite; server: IServer
     <>
       <List searchBarPlaceholder="Search sites...">
         <List.Section title={`Site Commands (${current.name})`}>
+          <List.Item
```

Comment:
Can you put this second?

Reply:
Sure

Reply:
Well I should have asked if you think it's more useful at the top. Personally I mainly use the deploy exclusively, but I wonder if I'm the only one. What do you think?

Reply:
We have gitlab webhooks set up that run deploy automatically so personally I would rarely use the deploy script, but I frequently open the site in the Forge dashboard to check/modify a .env value or perform some other task

Reply:
I'm thinking the Open command may be more popular. I think that's how the Alfred workflow worked. If you want to add it back to how you had it then I think that's fine. I don't think it should be specific to my personal use.

Reply:
Done—same on the quick links too